examples/demo/modules/blog/handler/blog_user_handler.e

```diff
@@ -30,71 +35,76 @@ feature -- HTTP Methods
 	do_get (req: WSF_REQUEST; res: WSF_RESPONSE)
 			-- <Precursor>
 		local
-			l_error: GENERIC_VIEW_CMS_RESPONSE
+			l_error: NOT_FOUND_ERROR_CMS_RESPONSE
 		do
-			-- Check if userID valid
-			if user_valid (req) then
-				user := load_user(req)
-				-- Output the results, similar as in the blog hanlder (but with other queries)
-				precursor(req, res)
+			user := Void
+			if attached user_from_request (req) as l_user then
+				user := l_user
+					-- Output the results, similar as in the blog hanlder (but with other queries)
+				Precursor (req, res)
 			else
-				-- Throw a bad request error because the user is not valid
+					-- Throw a bad request error because the user is not valid
 				create l_error.make (req, res, api)
-				l_error.set_main_content ("<h1>Error</h1>User with id " + user_path_parameter(req).out + " doesn't exist!")
+				l_error.set_main_content ("<h1>Error</h1>User with id " + user_id_path_parameter (req).out + " doesn't exist!")
 				l_error.execute
 			end
-
 		end
 
 feature -- Query
 
 	user_valid (req: WSF_REQUEST) : BOOLEAN
-			-- Returns true if a valid user id is given and a user with this id exists; otherwise returns false.
+			-- Returns true if a valid user id is given and a user with this id exists,
+			-- otherwise returns false.
 		local
 			user_id: INTEGER_32
 		do
-
-			user_id := user_path_parameter(req)
+			user_id := user_id_path_parameter (req)
 
 			if user_id <= 0 then
-				--Given user id is not valid
-				Result := false
+					-- Given user id is not valid
+				Result := False
 			else
-				--Check if user with user_id exists
+					--Check if user with user_id exists
 				Result := api.user_api.user_by_id (user_id) /= Void
 			end
 		end
 
-	load_user (req: WSF_REQUEST) :  detachable CMS_USER
```

Comment:
load_user was a bad Eiffel name for a function
we could used "user_loaded" or similar
here I choosed  "user_from_request" as it is quite descriptive and still not too long